cmd/res/profiles/llrp.impinj.profile.yaml

```diff
@@ -0,0 +1,153 @@
+name: "LLRP-Impinj-Profile"
+manufacturer: "Impinj"
+labels:
+  - "LLRP"
+  - "RFID"
+  - "Impinj"
+description: "A device profile with added Impinj support."
+
+deviceResources:
+  - 
+    name: "ImpinjCustomExtensionMessage"
+    properties:
+      valueType: "String"
+      readWrite: "RW"
```

Comment:
I alluded to this in the App service review, but I think this can just be "W".

Reply:
changed this to W